src/pages/iou/IOUModal.js

```diff
@@ -234,25 +236,39 @@ class IOUModal extends Component {
     }
 
     /**
-     * @param {Array} [splits]
+     * Update the payment type
+     *
+     * @param {String} paymentType
      */
-    createTransaction(splits) {
-        const reportID = lodashGet(this.props, 'route.params.reportID', '');
+    updatePaymentType(paymentType) {
+        this.setState({
+            paymentType,
+        });
+    }
 
-        // Only splits from a group DM has a reportID
-        // Check if reportID is a number
-        if (splits && CONST.REGEX.NUMBER.test(reportID)) {
-            createIOUSplitGroup({
-                comment: this.state.comment,
+    /**
+     * Create the IOU transaction
+     *
+     * @param {Array} [splits]
+     */
+    confirm(splits) {
+        if (this.state.iouType === CONST.IOU.IOU_TYPE.SEND) {
+            payIOUReport({
+                chatReportID: this.props.route.params.reportID,
+                reportID: 0,
+                paymentMethodType: this.state.paymentType,
 
                 // should send in cents to API
                 amount: Math.round(this.state.amount * 100),
                 currency: this.props.iou.selectedCurrencyCode,
-                splits,
-                reportID,
+                submitterPayPalMeAddress: this.state.participants[0].participantsList[0].paypalMeAddress,
+                submitterPhoneNumber: this.state.participants[0].participantsList[0].phoneNumber,
```

Comment:
Testing this PR and I'm not seeing the payPalMeAddress show up in the IOUModal component. I _think_ we'd need to add the `payPalMeAddress` and `phoneNumber` to the mapping of the `participantsWithDetails` const on line 110 above to get this to work, but correct me if I'm wrong

@mountiny